scripts/collection_from_items.py

```diff
@@ -42,6 +37,13 @@ def main() -> None:
     parser.add_argument(
         "--location", dest="location", help="Optional Location of dataset, e.g.- Hutt City", type=str, required=False
     )
+    parser.add_argument(
+        "--geographic-desc",
+        dest="geographic_description",
+        help="Optional Location Name, e.g.- South Island",
```

Comment:
is the `-` needed? is `e.g.-` seems odd is this a convention im just not aware of?